SMITHY_VERSION

```diff
@@ -0,0 +1 @@
+1.26.1
```

Comment:
Modifying how we manage dependencies to make your CI task easier isn't a great solution

I think you need to figure out a way to handle your upstream CI without modifying repositories. There are quite a few solutions that wouldn't require changes such as simple sed command, composite builds, property override when invoking gradle (`.gradlew -PsmithyVersion=`), etc. 

As is I don't think we want to accept this PR.